client/src/app/components/home/home.component.ts

```diff
@@ -18,7 +18,7 @@ export class HomeComponent implements OnInit, OnDestroy {
   isLiked: boolean = false
 
   forecastData:any;
-  isLoading: boolean = false;
+  isLoading: boolean = true;
```

Comment:
make loading effect true in home page because it needs to pop up loading effect before load the home page